--- a/target/linux/ramips/mt76x8/base-files/etc/board.d/02_network
+++ b/target/linux/ramips/mt76x8/base-files/etc/board.d/02_network
@@ -55,6 +55,7 @@
 	tplink,archer-c50-v3|\
 	tplink,archer-c50-v4|\
 	tplink,archer-c50-v6|\
+	tplink,ec220-f5-v1|\
 	tplink,tl-mr3420-v5|\
 	tplink,tl-wr840n-v4|\
 	tplink,tl-wr840n-v5|\
@@ -363,7 +364,8 @@
 		;;
 	tplink,archer-c20-v5|\
 	tplink,archer-c50-v4|\
-	tplink,archer-c50-v6)
+	tplink,archer-c50-v6|\
+	tplink,ec220-f5-v1)
 		wan_mac=$(macaddr_add "$(mtd_get_mac_binary rom 0xf100)" 1)
 		;;
 	wavlink,wl-wn570ha1|\
